operator/gitops/argocd/pipeline-service/kustomization.yaml

```diff
@@ -4,7 +4,6 @@ kind: Kustomization
 
 resources:
   - openshift-pipelines
-  - pipelines-as-code
```

Comment:
question in https://github.com/openshift-pipelines/pipeline-service/pull/618#issuecomment-1522047401 answered :-)

are you doing chains in a separate PR for isolation purposes @Roming22 ?

thanks

Reply:
Correct.